cmd/controller-manager/OWNERS

```diff
@@ -0,0 +1,66 @@
+approvers:
+- deads2k
+- lavalamp
+- mikedanese
```

Comment:
Mike is at a higher level, he doesn't need to be listed here.

Reply:
same for Daniel. Let's use the inner circe of people working the plumbing: the two of us and cheftako. Same list for review.

Reply:
done